src/components/Loader/Loader.style.tsx

```diff
@@ -1,30 +1,32 @@
-import styled from 'styled-components';
-
-export const getDotsLayout = (delay: number, animation: string, left?: number) => `
-    left: ${`${left}px` || ''};
-    width: 6px;
-    height: 6px;
-    border-radius: 5px;
-    background-color: #cfcfcf;
-    color: #cfcfcf;
-    animation: ${animation};
-    animation-delay: ${delay}s;
+import { css, SerializedStyles } from '@emotion/core';
+
+import { Theme } from 'theme';
+
+export const getDotsLayout = (delay: number, animation: string, theme: Theme, left?: number) => css`
+  left: ${`${left}px` || ''};
```

Comment:
this is wrong. This will return `undefinedpx` in all cases. Also the braces are wrong

solution
`left && `${left}px`;`
otherwise it will be undefined and will not set left at all

Reply:
This is really really wrong, I forgot to change it at some point. It actually worked though, in all cases :P